scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,118 @@
+"""Module to generate a column map to their corresponding statistical variable defintions"""
+
+import os
+import json
+import pandas as pd
+
+#TODO:Remove this import and make it from ~/data/utils/ directory when it is available
+os.system('curl https://raw.githubusercontent.com/Abilityguy/data/dcid_gen/util/statvar_dcid_generator.py -LO statvar_dcid_generator.py')
+import statvar_dcid_generator
+
+dcid_gen = statvar_dcid_generator.StatVarDcidGenerator()
+
+
+def generate(specDict, columnList):
+  """Wrapper function for generateColMapBase class to generate column map"""
+  col_map_obj = generateColMapBase(specDict=specDict, columnList=columnList)
+  return col_map_obj._generate_stat_vars_from_spec()
+
+class generateColMapBase:
+  """module to generate a column map given a list of columns of the dataset and a JSON Spec"""
+  def __init__(self, specDict=None, columnList=None):
+    """module init """
+    self.features = specDict
+    self.column_list = columnList
+    self.column_map = {}
+
+  def _generate_stat_vars_from_spec(self):
+    """generates stat_var nodes for each column in column list and returns a dictionary of columns with their respective stat_var nodes"""
+    # for each column generate the definition of their respective statistical variable node
+    for col in self.column_list:
+      if 'ignoreColumns' in self.features and col not in self.features['ignoreColumns']:
+          self.column_map[col] = self._column_to_statvar(col)
+    #TODO: Enable validating the generate columnMap: _valid_col_map():
```

Comment:
nit: consistent spacing.

also, please run yapf/lint - https://screenshot.googleplex.com/7hgXcwPiim2tQwJ

Reply:
I shall do this.